cardano-cli/src/Cardano/CLI/Run/Key.hs

```diff
@@ -317,10 +322,58 @@ runExtendedSigningKeyFromMnemonicCmd
               writeTextFile sKeyPath $
                 serialiseToBech32 skey
 
-readMnemonic :: Cmd.MnemonicSource -> ExceptT KeyCmdError IO [Text]
-readMnemonic (Cmd.MnemonicFromFile filePath) = do
-  fileText <- firstExceptT KeyCmdReadMnemonicFileError $ except =<< readTextFile filePath
-  return $ map T.pack $ words $ T.unpack fileText
+    readMnemonic :: Cmd.MnemonicSource -> ExceptT KeyCmdError IO [Text]
+    readMnemonic (Cmd.MnemonicFromFile filePath) = do
+      fileText <- firstExceptT KeyCmdReadMnemonicFileError $ except =<< readTextFile filePath
+      return $ map T.pack $ words $ T.unpack fileText
+    readMnemonic Cmd.MnemonicFromInteractivePrompt =
+      liftIO $ do
+        putStrLn ""
```

Comment:
`unlines` would be a good refactor here

Reply:
Addressed here: https://github.com/IntersectMBO/cardano-cli/pull/975/commits/51c0c94463cadd1551cfce9fd5e0cff8ea9adafe